app/controllers/qc_infos_controller.rb

```diff
@@ -0,0 +1,13 @@
+class QcInfosController < ApplicationController
+  include Concerns::ControllerHelper
+
+  def edit
+    @qc_info = QcInfo.find(params[:id])
```

Comment:
It raises an exception.
See https://api.rubyonrails.org/v4.2/classes/ActiveRecord/FinderMethods.html#method-i-find

Reply:
I think that we don't have a proper exceptions handling in the entire platform.  Something like storing a log in Sentry and showing a nice message to the user.  Right now, I guess we're showing a 500 internal server error?  

I'd create a tech debt issue and move forward.  Perhaps we can start thinking about investing a full week in paying debt.

Reply:
It's more than an acceptable behavior, it's idiomatic. It's similar to how `Hash#[]` will raise an IndexError by default: we expect a happy path scenario.

In a hopeful world, it shouldn't fail. In the real world, it will fail, and CDx is missing a few `rescue_from` callbacks in `ApplicationController` (and/or others) to handle those exceptions properly. For example:

```ruby
rescue_from ActiveRecord::NotFound do
  respond_to do |format|
    format.json { render json: {error: "not_found"}, status: 404 }
    format.html { render "errors/not_found", status: 404 }
  end
end
```

This is just out of the top of my head, and what we usually do in Rails. We should check what's the usual format of reporting errors in CDx.

In the meantime, it logs an error to Sentry in staging (I don't know if Sentry is configured in production) and probably presents a generic "500 Internal Server Error" message.